### ISSUE TITLE
Windows 7 error

### ISSUE THREAD

**Issue by username99987** [kotas/gulp-tsc#3]
Hello.

I am having strange error when trying to compile any .ts via gulp-tsc (tried both 0.0.2 and 0.0.3)
Here is the output:

``` bash
[gulp] Failed to compile TypeScript: Error: Command failed:
module.js:340
    throw err;
          ^
Error: Cannot find module '...\Documents\frontend\tool\'C:\Program'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Function.Module.runMain (module.js:497:10)
    at startup (node.js:119:16)
    at node.js:902:3
```

Seems like the module path is screwed.
However, in Ubuntu it works.
Any idea for workaround?
Thanks.

**Comment by kotas** [comment]
@onion-soup Thank you for your report!

I have just published v0.0.4 to fix this problem.

It seems to be a problem by a file path containing whitespace.

I tested v0.0.4 on my Windows machine, but I'm not 100% sure that fixes your problem.  So could you try it and tell me if it still has issues. Thank you!

**Comment by username99987** [comment]
Thanks for a quick response!
Problem is solved! :+1:
